docker-compose.gpu.yml

```diff
@@ -12,11 +12,9 @@ services:
               count: 1
               capabilities: [gpu]
     environment:
-      - ASR_MODEL=small
       - ASR_ENGINE=faster_whisper
     ports:
       - "9000:9000"
-    restart: on-failure
```

Comment:
Removing this because it has the side effect of always starting the container at boot (even if it was not running at shutdown), and additionally leads to infinite loops when the permissions aren't set up right. The docs explain how to customize this value.